src/renderer/component/common/tooltip.jsx

```diff
@@ -33,11 +32,11 @@ class ToolTip extends React.PureComponent<Props, State> {
   }
 
   getVisibility = () => {
-    if (!this.tooltip.current) {
+    if (!this.tooltip) {
```

Comment:
can make tooltip local with `let tooltip = this.tooltip;` at the top, will reduce object prop lookups